realm/realm-library/src/main/java/io/realm/RealmCache.java

```diff
@@ -164,17 +193,15 @@ private RealmCache(RealmConfiguration config) {
             if (realmClass == Realm.class && refAndCount.globalCount == 0) {
```

Comment:
You are right, since `localCount` will be reset when `release()`. But i actually prefer the current code, it is easier to read without knowing `localCount` will be reset :P